Is7326.h

```diff
@@ -1,54 +1,104 @@
-#ifndef __IS31IO7326_H__
-#define __IS31IO7326_H__
+#pragma once
 
 #include <Arduino.h>
 
-typedef struct {
-  byte ad01;
-  boolean down;
-  byte key;
-} key_t;
+#define IS7326_KEY_DOWN_MASK 0x40
+#define IS7326_KEY_CODE_MASK 0x3F
```

Comment:
I'm going to sleep on this, but despite the multiple-return-value API, I kinda prefer the `struct` approach. The old-school-systems-C "negative on failure, non-negative result on success" pattern is a pretty outdated, and awkward to use in practice (e.g. the example project).

If this API reverts back to using a struct, it should be something more accurate to the actual type (the old struct definition was ⅔ wasted space), like:

```
typedef struct {
    uint8_t key:6,
            down:1,
            reserved:1;
} is7326_key_t;
```

When packed, the above struct is 1 byte and can be trivially used in a union to populate the struct directly from data received over I²C, in the manner of the [is31io7326_clone](https://github.com/numist/is31io7326_clone/blob/master/firmware/is31io7326.h#L7-L15) project.